custom_dc/env.list

```diff
@@ -1,7 +1,20 @@
+### API Keys ###
+
 # API key for accessing the base Data Commons API.
 DC_API_KEY=
 # API key for accessing Maps and Places APIs for exploration tools.
 MAPS_API_KEY=
+
+### Directories ###
+
+# Input directory in Google Cloud Storage or local drive, containing files to be processed.
```

Comment:
Please add a line about how to specify the directory format for GCS, i.e. gs://<GCS bucket>/<directory>

Reply:
Done.